README.md

```diff
@@ -2,23 +2,72 @@
 
 # [ghuser.io](https://github.com/ghuser-io/ghuser.io)'s database
 
-It is here to cache GitHub's data. During the prototyping phrase, it's just a set of
```

Comment:
For me it was relevant that this won't remain a set of json files forever

Reply:
To me it looks like this belongs to a backlog or contribution section, not as the first sentence of the README.